youtube_dl/extractor/epidemicsound.py

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import unified_timestamp
```

Comment:
To match the other suggestion:
```suggestion
from ..utils import (
    ExtractorError,
    float_or_none,
    txt_or_none,
    unified_timestamp,
    url_or_none,
)